HOMEWORK_6/EX2.CPP

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+
+int main()
+{
+    int elementNumber;
+    int result;
+    int index = 0;
+    int firstValue = 0;
+    int secondValue = 1;
+
+    std::cout<<"Enter the element number: ";
+    std::cin>>elementNumber;
+    
+    if(elementNumber > 0)
+    {
+        do
+        {
+            result = firstValue + secondValue;
+
+            if(index > 0)
```

Comment:
I just don't feel like this condition is a must-have... I can think of simpler solution

Reply:
I haven't no idea, how to do it(

Reply:
int first = 0;
int  second = 1;
int summ = 0;

    if (elementNubmer == 0)
        return first;

    for (int i = 2; i <= elementNubmer; i++) {
        summ = first + second;
        first = second;
        second = summ;
    }
    return second;
}